### PR TITLE
Add documentations on how to setup /etc/subuid and /etc/subgid

### PR DESCRIPTION
**Pull request by rhatdan**
Signed-off-by: Daniel J Walsh <dwalsh@redhat.com>

### CODE REVIEW

**Comment by rhatdan** [comment]
@giuseppe PTAL

**Comment by rhatdan** [comment]
Also pulled in https://github.com/containers/storage/pull/205

**Comment by rhatdan** [comment]
Fixes https://github.com/projectatomic/libpod/issues/1182

**Comment by vrothberg** [comment]
LGTM

In addition to the improved error messages, this is making it much more user-friendly.

File: docs/podman.1.md [hunk 1]
```diff
@@ -171,6 +171,17 @@ Containers created by a non-root user are not visible to other users and are not
 
 It is required to have multiple uids/gids set for an user.  Be sure the user is present in the files `/etc/subuid` and `/etc/subgid`.
 
+If you have a recent version of usermod, you can execute the following
+commands to add the ranges to the files
+
+	$ sudo usermod --add-subuids 10000-65536 USERNAME
```
Comment (giuseppe) [inline docs/podman.1.md]: I checked and it adds less users than the manual command below (the second number is the maximum ID, not the length of the range).  Let's change it to `10000-75535`

**Comment by giuseppe** [comment]
LGTM

@rh-atomic-bot r+

**Pull request closed without merging**